src/components/ec-filter-popover/ec-filter-popover.spec.js

```diff
@@ -0,0 +1,51 @@
+import { shallowMount } from '@vue/test-utils';
+import EcFilterPopover from './ec-filter-popover.vue';
+import { withMockedConsole } from '../../../tests/utils/console';
+
+const label = 'Test label';
+
+function shallowMountEcFilterPopover(props) {
+  return shallowMount(EcFilterPopover, {
```

Comment:
We should use shallowMount only as a last option, in this case, I'd rather use mount() and stub EcPopover instead. In the future, this component will have ec-number-badge and that component will be stubbed by shallowMount too, so we cannot test what it renders here. With mount and stubbed EcPopover, we will not have that problem.